template/.github/workflows/build.yml.j2

```diff
@@ -379,3 +379,70 @@ jobs:
         run: ./preflight-linux-amd64 check container "$IMAGE_TAG" > preflight.out
       - name: "Passed?"
         run: '[ "$(./preflight-linux-amd64 check container "$IMAGE_TAG" | jq -r .passed)" == true ]'
+
+  openshift_tests:
+    name: Run integration tests
+    if: ${{ !github.event.pull_request.head.repo.fork }}
+    needs:
+      - package_and_publish
+    strategy:
+      fail-fast: false
+      matrix:
```

Comment:
What happens if we want to run tests on other clusters? Just extend the matrix here and rename `openshift_tests` to e.g. `cluster_tests`?

Reply:
Renamed: https://github.com/stackabletech/operator-templating/pull/333/commits/28a7a6c8f28f367c6fcb280cdf8ea61c971b6c7d